Mathlib/Order/KrullDimension.lean

```diff
@@ -395,22 +395,55 @@ lemma krullDim_eq_of_orderIso (f : α ≃o β) : krullDim α = krullDim β :=
 
 /--
 The Krull dimension is the supremum of the elements' heights.
+
+See `krullDim_eq_iSup_height` for the more general, but less convenient version without the
+`Nonempty` requirement.
```

Comment:
Can you explain the precise difference between the two statements? It's hard to make out from just staring at the code